2022-01-24 09:21:26,458 INFO    MainThread:235217 [wandb_setup.py:_flush():71] setting env: {}
2022-01-24 09:21:26,458 INFO    MainThread:235217 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-24 09:21:26,458 INFO    MainThread:235217 [wandb_init.py:_log_setup():371] Logging user logs to /storage/projects/pgrad-thesis/wandb/run-20220124_092126-3dj11kkb/logs/debug.log
2022-01-24 09:21:26,458 INFO    MainThread:235217 [wandb_init.py:_log_setup():372] Logging internal logs to /storage/projects/pgrad-thesis/wandb/run-20220124_092126-3dj11kkb/logs/debug-internal.log
2022-01-24 09:21:26,459 INFO    MainThread:235217 [wandb_init.py:init():404] calling init triggers
2022-01-24 09:21:26,459 INFO    MainThread:235217 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-24 09:21:26,459 INFO    MainThread:235217 [wandb_init.py:init():460] starting backend
2022-01-24 09:21:26,459 INFO    MainThread:235217 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-24 09:21:26,465 INFO    MainThread:235217 [backend.py:ensure_launched():216] starting backend process...
2022-01-24 09:21:26,469 INFO    MainThread:235217 [backend.py:ensure_launched():221] started backend process with pid: 235279
2022-01-24 09:21:26,469 INFO    MainThread:235217 [wandb_init.py:init():469] backend started and connected
2022-01-24 09:21:26,472 INFO    MainThread:235217 [wandb_init.py:init():533] updated telemetry
2022-01-24 09:21:26,484 INFO    MainThread:235217 [wandb_init.py:init():563] communicating current version
2022-01-24 09:21:27,044 INFO    MainThread:235217 [wandb_init.py:init():568] got version response 
2022-01-24 09:21:27,044 INFO    MainThread:235217 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-24 09:21:27,340 INFO    MainThread:235217 [wandb_init.py:init():606] starting run threads in backend
2022-01-24 09:21:29,879 INFO    MainThread:235217 [wandb_run.py:_console_start():1810] atexit reg
2022-01-24 09:21:29,879 INFO    MainThread:235217 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-24 09:21:29,879 INFO    MainThread:235217 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-24 09:21:29,880 INFO    MainThread:235217 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-24 09:21:29,880 INFO    MainThread:235217 [wandb_init.py:init():633] run started, returning control to user process
2022-01-24 09:21:31,022 INFO    MainThread:235217 [wandb_watch.py:watch():43] Watching
2022-01-24 09:21:31,022 ERROR   MainThread:235217 [util.py:get_module():187] Error importing optional module torch
Traceback (most recent call last):
  File "/home/aadi/miniconda3/envs/pgrad-thesis/lib/python3.8/site-packages/wandb/util.py", line 182, in get_module
    return import_module(name)
  File "/home/aadi/miniconda3/envs/pgrad-thesis/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'torch'
2022-01-24 09:21:31,023 INFO    MainThread:235217 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-24 09:21:31,024 INFO    MainThread:235217 [wandb_run.py:_restore():1752] restore
2022-01-24 09:21:33,520 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 785
  total_bytes: 785
}

2022-01-24 09:21:33,875 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 785
  total_bytes: 785
}

2022-01-24 09:21:34,347 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 785
  total_bytes: 39398
}

2022-01-24 09:21:34,448 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 785
  total_bytes: 51230
}

2022-01-24 09:21:34,549 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 785
  total_bytes: 51230
}

2022-01-24 09:21:34,650 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 785
  total_bytes: 51230
}

2022-01-24 09:21:34,751 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 785
  total_bytes: 51230
}

2022-01-24 09:21:34,852 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 41905
  total_bytes: 51230
}

2022-01-24 09:21:34,954 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 51230
  total_bytes: 51230
}

2022-01-24 09:21:35,055 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 51230
  total_bytes: 51230
}

2022-01-24 09:21:35,157 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 51230
  total_bytes: 51230
}

2022-01-24 09:21:35,258 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 51230
  total_bytes: 51230
}

2022-01-24 09:21:35,359 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 51230
  total_bytes: 51230
}

2022-01-24 09:21:35,536 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 51230
  total_bytes: 51230
}

2022-01-24 09:21:36,173 INFO    MainThread:235217 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 51230
  total_bytes: 51230
}
local_info {
}

2022-01-24 09:21:37,355 INFO    MainThread:235217 [wandb_run.py:_append_files():2180] logging synced files
